src/main/java/com/amazon/dlic/auth/http/saml/AuthTokenProcessorHandler.java

```diff
@@ -118,18 +122,19 @@ class AuthTokenProcessorHandler {
     }
 
     @SuppressWarnings("removal")
-    BytesRestResponse handle(RestRequest restRequest) throws Exception {
+    boolean handle(RestRequest restRequest, RestChannel restChannel) throws Exception {
```

Comment:
I still need to do a deeper dive into your changes here, I might have reverted them too quickly.

This class it was written in such a way that it wouldn't require authentication to process the request which is the ACS endpoint request for SAML, but frankly it should be registered as our other http handlers with an allowance not to require authentication, it would greatly clean up these classes if this was 'unhooked' from the reauth flow.  I tried to avoid doing too many refactors at once.

Reply:
I've reincluded your changes, but I changed the format to `SecurityResponse` to keep using a more generic interface